docs/ui.md

```diff
@@ -27,83 +27,10 @@ These elements are defined in the manifest.xml file:
 </connector-plugin>
 ```
 
-## Vendor Attributes (Custom Fields)
```

Comment:
should this be in the 2020.1 branch?

Reply:
Reverted the revert, looks like everything is as expected now.